inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -17,6 +17,9 @@
 #include <ngraph/pass/manager.hpp>
 #include <ngraph/type/bfloat16.hpp>
 
+#include "transformations/serialize.hpp"
```

Comment:
do we need it in header file? I suppose it's used only in cpp

Reply:
fixed